src/sqlfluff/core/linter/runner.py

```diff
@@ -211,3 +204,28 @@ def __init__(self, ee):
     def reraise(self):
         """Reraise the encapsulated exception."""
         raise self.ee.with_traceback(self.tb)
+
+
+def get_runner(
+    linter,
+    config,
+    parallel: int,
+    allow_process_parallelism: bool = True,
+) -> BaseRunner:
+    """Generate a runner instance based on parallel and sytem configuration."""
+    if parallel > 1 and sys.version_info > (3, 7):
```

Comment:
I noted this on the other PR as well -- I think the version check should be `sys.version_info > (3, 7)`. (Minor bug.)

Reply:
@barrywhart I'm confused. The check is _already_ `(3, 7)`, did you mean it should be something else? I'd also love to add a comment saying _why_ the earlier versions aren't supported - do you know what the issue is with them?

Reply:
Scratch that.... saw your other comment. Will fix it there.